engine/templates_tests/templatedOutputError.yaml

```diff
@@ -0,0 +1,17 @@
+name: templatedOutputError
+description: A template with a templated output and base output
+title_format: "[test] Templated Output Error"
+auto_runnable: true
+steps:
+    stepOne:
+        description: first step
+        idempotent: true
+        retry_pattern: seconds
+        action:
+            type: echo
+        configuration:
+            key: value
+        base_output:
+            configuration: someOutput
+        templated_output:
```

Comment:
This template seems to indicate it should fail because of templated_output and base_output defined at the same time, but
* templated_output didn't exist anymore in the pull-request
* the template is failing because configuration is not indented properly and should be inside the action structure
* also, note that key: value should be inside the output struct.

I guess we should remove this file